src/main/java/javaslang/collection/List.java

```diff
@@ -1202,7 +1202,7 @@ default T peek() {
     }
 
     @Override
-    default List<T> unit(Iterable<? extends T> iterable) {
+    default List<T> cons(Iterable<? extends T> iterable) {
```

Comment:
@danieldietrich I'm not sure cons is a good term for this method.  Cons usually takes an explicit head and tail.